package.json

```diff
@@ -36,10 +37,22 @@
     "styled-components": "^4.3.2",
     "styled-icons": "^8.1.0",
     "typesafe-actions": "^4.4.2",
-    "typescript": "3.9.6",
+    "typescript": "^4.2.0",
     "url": "^0.11.0",
     "uuid": "^3.4.0",
-    "weak-map": "^1.0.5"
+    "weak-map": "^1.0.5",
+    "workbox-background-sync": "^5.1.3",
+    "workbox-broadcast-update": "^5.1.3",
+    "workbox-cacheable-response": "^5.1.3",
+    "workbox-core": "^5.1.3",
+    "workbox-expiration": "^5.1.3",
+    "workbox-google-analytics": "^5.1.3",
+    "workbox-navigation-preload": "^5.1.3",
+    "workbox-precaching": "^5.1.3",
+    "workbox-range-requests": "^5.1.3",
+    "workbox-routing": "^5.1.3",
+    "workbox-strategies": "^5.1.3",
+    "workbox-streams": "^5.1.3"
```

Comment:
i feel like some of these workbox plugins aren't used, are they?

Reply:
Indeed it seems like this. I've copied all of them from the template created with `yarn create react-app my-app --template cra-template-pwa-typescript` and they are not used there either 
I've removed them for now and we can check if they work

Reply:
After removing them service worker is not being registered. I'll revert this change

Reply:
Oh, it is actually not working after reverting. I think it may be related to the change about PUBLIC_URL:
```
if (!process.env.PUBLIC_URL) {
    return Promise.reject(
      new Error('service worker won\'t work if PUBLIC_URL is not defined.')
    );
  }
```

Reply:
I've ended up changing the `if` statement to `if (process.env.PUBLIC_URL === undefined) {` and removed these workbox plugins

Btw. `process.env.PUBLIC_URL` is set to an empty string on `rex-web.herokuapp.com`